examples/hooks/RecordHooks/conditionallyModifyField.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Sheet, Workbook, BooleanField, TextField } from '@flatfile/configure'
+import { SheetTester } from '../../../src/utils/testing/SheetTester'
+
+//this Data Hook can be used to update the contents of a cell based on whether or not another cell passes RegEx validation.
+//This method of setting fields at validations at the top of a sheet allows you to set all RegEx and messages in one easily referencable place.
+const FIELDS = {
+  accountId: {
+    regex: /^.{1,20}$/,
+    message: 'Account Number must be 20 characters or less',
+  },
+}
+
+const testSheet = new Sheet(
+  'Test Sheet',
+  {
+    accountId: TextField(),
+    accountStatus: BooleanField(),
+  },
+  {
+    recordCompute: (record) => {
+      //ensuring the accountId is returned as a string
+      const accountNum = record.get('accountId') as string
```

Comment:
@paddymul I needed to ensure I was returning the accountId field as a string here otherwise TypeScript was concerned that the TPrimitive type could be a number, boolean, string, or null. Is there a better way to ensure this is a string to be able to test properly?